rubrik_cdm/data_management.py

```diff
@@ -885,3 +886,81 @@ def get_sla_objects(self, sla, object_type, timeout=15):
                         sla, object_type))
 
             return vm_name_id
+
+    def time_in_range(self, start, end, x):
+        """Return true if x is in the range [start, end]"""
+        if start <= end:
+            return start <= x <= end
+        else:
+            return start <= x or x <= end
+
+    def sql_live_mount(self, sql_db, sql_instance, sql_host, clone_name, date, time, timeout=15):  # pylint: ignore
+        """Live Mount a database from a specified recovery point. 
+
+        Arguments:
+            sql_db {str} -- The name of the database to Live Mount.
+            sql_instance {str} -- The SQL instance name for the database to Live Mount.
+            sql_host {str} -- The SQL Host of the database/instance to Live Mount.
+            clone_name {str} -- The name given to the Live Mounted database i.e. clone name, AdventureWorks_Clone.
+            date {str} -- The date of the recovery_point you wish to Live Mount formated as `Month-Day-Year` (ex: 1-15-2014). 
+            time {str} -- The time of the recovery_point you wish to Live Mount formated formated as `Hour:Minute AM/PM` (ex: 1:30 AM).
+
+        Keyword Arguments:
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            dict -- The full response of `POST /v1/mssql/db/{id}/mount`.
+        """
+        
+        mssql_host_id = self.object_id(sql_host, 'physical_host', timeout=timeout)
+
+        mssql_instance = self.get(
+            'v1', '/mssql/instance?primary_cluster_id=local&root_id={}'.format(mssql_host_id), timeout)
+
+        for instance in mssql_instance['data']:
+            if instance['name'] == sql_instance:
+                sql_instance_id = instance['id']
+
+        mssql_db = self.get('v1', '/mssql/db?primary_cluster_id=local&instance_id={}'.format(sql_instance_id), timeout)
```

Comment:
Add a logging message for 

`mssql_db = self.get('v1', '/mssql/db?primary_cluster_id=local&instance_id={}'.format(sql_instance_id), timeout)`


that describes what it is doing